src/main/java/com/iota/iri/service/dto/GetNeighborsResponse.java

```diff
@@ -2,40 +2,86 @@
 
 import java.util.List;
 
+/**
+ * This class represents the core API request 'getNeighbors'.
+ * Returns the set of neighbors you are connected with, as well as their activity count. 
+ * The activity counter is reset after restarting IRI.
+ *
+ **/
 public class GetNeighborsResponse extends AbstractResponse {
 
     private Neighbor[] neighbors;
 
+    /**
+     * The list of neighbors, including the following stats: 
+     *  address, connectionType,
+     *  numberOfAllTransactions, numberOfRandomTransactionRequests, 
+     *  numberOfNewTransactions, numberOfInvalidTransactions, numberOfSentTransactions
+     * 
+     * @see {@link com.iota.iri.service.dto.GetNeighborsResponse.Neighbor}
+     * @return the neighbors
```

Comment:
Thinking about it again:
won't it be better to simply do:
`@return Array of {@link com.iota.iri.service.dto.GetNeighborsResponse.Neighbor} elements.`
?

Reply:
Each response has more then 1 element, so an array of neighbors wont work. There is always a duration: [time].

I loop over all public getters WITH javadoc on each Response
Array is based on the getter return type being more than 1 dimensional, or a form of java.util.List